blueprints/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/formula_5_10.py

```diff
@@ -0,0 +1,53 @@
+"""Formula 5.10 from NEN-EN 1993-5:2008 Chapter 5 - Ultimate limit state."""
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008 import NEN_EN_1993_5_2008
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import DIMENSIONLESS, KN
+from blueprints.validations import raise_if_less_or_equal_to_zero
+
+
+class Form5Dot10ReductionFactorShear(Formula):
+    """Class representing formula 5.10 for reduction factor for shear resistance of the cross-section."""
+
+    label = "5.10"
+    source_document = NEN_EN_1993_5_2008
+
+    def __init__(
+        self,
+        v_ed: KN,
+        v_pl_rd: KN,
+    ) -> None:
+        """[:math:`ρ`] Calculate the reduction factor for shear resistance of the cross-section [-].
+
+        NEN-EN 1993-5:2008(E) art.5.2.2(9) - Formula (5.10)
+
+        Parameters
+        ----------
+        v_ed : KN
+            [:math:`V_{Ed}`] Design shear force in [:math:`kN`].
+        v_pl_rd : KN
+            [:math:`V_{pl,rd}`] Plastic shear resistance in [:math:`kN`].
+        """
+        super().__init__()
+        self.v_ed: float = v_ed
+        self.v_pl_rd: float = v_pl_rd
+
+    @staticmethod
+    def _evaluate(
+        v_ed: KN,
+        v_pl_rd: KN,
+    ) -> DIMENSIONLESS:
+        """Evaluates the formula for reduction factor for shear resistance."""
+        raise_if_less_or_equal_to_zero(v_ed=v_ed, v_pl_rd=v_pl_rd)
+        return (2 * v_ed / v_pl_rd - 1) ** 2
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for formula 5.10."""
+        return LatexFormula(
+            return_symbol=r"\rho",
+            result=str(self),
+            equation=r"\left(2 \cdot \frac{V_{Ed}}{V_{pl,Rd}} - 1\right)^2",
+            numeric_equation=rf"\left(2 \cdot \frac{{{self.v_ed}}}{{{self.v_pl_rd}}} - 1\right)^2",
```

Comment:
why not use the latex_fraction function here?

Reply:
Sharp observation! There isn't a specific reason, but for the sake of simplicity, let's leave this one as it is.